go.mod

```diff
@@ -2,13 +2,14 @@ module local/build
 
 go 1.17
```

Comment:
Could you update this to Go 1.18?

Reply:
Fixed at https://github.com/hazelcast/hazelcast-cloud-go-sample-client/pull/14/commits/8780cc95bbc6f29f809d266101019e026a081ca3